src/server/game/Entities/Player/Player.cpp

```diff
@@ -21113,15 +21113,16 @@ bool Player::ActivateTaxiPathTo(std::vector<uint32> const& nodes, Creature* npc
         return false;
     }
 
-    // check node starting pos data set case if provided
+	// check node starting pos data set case if provided
```

Comment:
Please remove this change, it is unnecessary. TC source does not use TABs.
GitHub will use 1 TAB = 8 spaces in preview. Correct indent is 4 spaces.